.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We should exclude this file from the PR

Reply:
After removing .vscode folder, could you please add it to the `.gitignore` file?
It's better to avoid that IDE configuration files are committed accidentally 🥲